op-batcher/batcher/batch_submitter.go

```diff
@@ -36,9 +36,10 @@ func Main(version string, cliCtx *cli.Context) error {
 	}
 
 	l := oplog.NewLogger(cfg.LogConfig)
+	m := metrics.NewMetrics("default")
```

Comment:
When would we change "default" to something else?

Reply:
I just copied that from all other metrics impls. Wondering myself 😅

Reply:
Ok sounds good.